src/extension/node/extension_node.ts

```diff
@@ -83,6 +85,31 @@ export function activate(context: vscode.ExtensionContext): void {
     )
   );
 
+  const provider = new HelpViewProvider(context.extensionUri);
+
+  context.subscriptions.push(
+    vscode.window.onDidChangeTextEditorSelection(
+      (e: vscode.TextEditorSelectionChangeEvent) => {
+        const document = e.textEditor.document;
+        if (document.languageId === 'malloy') {
+          const parse = Malloy.parse({source: document.getText()});
+          const highlight = highlightForPosition(
+            parse.highlights,
+            e.selections[0].start
+          );
+
+          if (highlight) {
+            provider.showHelpFor(highlight.type);
+          }
+        }
+      }
+    )
+  );
+
+  context.subscriptions.push(
+    vscode.window.registerWebviewViewProvider('malloyHelp', provider)
+  );
+
```

Comment:
With the addition of browser support, there are now two extension bodies, `extension_node.ts` and `extension_browser.ts`, we want this to work with both. You should be able to move this to `setupSubscriptions()` and it will get picked up for both.